compiler/p/codegen/BinaryEvaluator.cpp

```diff
@@ -2372,6 +2372,8 @@ TR::Register *lrem64Evaluator(TR::Node *node, TR::CodeGenerator *cg)
    int64_t divisor = 0;
    TR::Compilation * comp = cg->comp();
 
+   TR_ASSERT(node->getOpCodeValue() != TR::lurem, "TR::ludiv is not impelemented yet for 64-bit target\n");
```

Comment:
spelling:   implemented

Reply:
For once, this is not my spelling error 😆 ... but I'll fix it.